xml/System/Console.xml

```diff
@@ -1208,6 +1208,7 @@
 
 ## Remarks
  A get operation for a Windows-based application, in which a console does not exist, returns <xref:System.ConsoleColor.Gray>.
+ On Linux, because there is no general mechanism to fetch the current Console colors, <xref:System.Console.ForegroundColor%2A> will return `(ConsoleColor)-1` until it has been set.
```

Comment:
same as above:

```suggestion
  Unix systems don't provide any general mechanism to fetch the current console colors. Because of that, <xref:System.Console.ForegroundColor%2A> returns `(ConsoleColor)-1` until it is set in explicit way (using the setter).
```